Tests/TestingMacrosTests/TestDeclarationMacroTests.swift

```diff
@@ -158,6 +154,91 @@ struct TestDeclarationMacroTests {
     }
   }
 
+  @Test("Error diagnostics which include fix-its emitted on API misuse", arguments: [
+    // 'Test' attribute must specify arguments to parameterized test functions.
+    "@Test func f(i: Int) {}":
+      (
+        message: "Attribute 'Test' must specify arguments when used with function 'f(i:)'",
+        fixIts: [
+          ExpectedFixIt(
+            message: "Add 'arguments:' with one collection",
+            changes: [.replace(oldSourceCode: "@Test ", newSourceCode: "@Test(arguments: \(editorPlaceholder(forType: "[Int]"))) ")]
```

Comment:
You should be able to pass an instance of `EditorPlaceholderExprSyntax` here and get it stringified correctly, no?

Reply:
Nice simplification, thanks